extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -401,7 +438,15 @@ class MembershipsButtonEdit extends Component {
 											'jetpack'
 										) }
 									</p>
-									<Button isDefault isLarge href={ connectURL } target="_blank">
+									<Button
+										isDefault
+										isLarge
+										disabled={ ! stripeConnectUrl }
+										href={ stripeConnectUrl }
+										target={ stripeConnectTarget }
+										rel={ stripeConnectTarget === '_blank' ? 'noopener noreferrer' : undefined }
+										onClick={ this.props.autosaveAndNavigateToConnection }
```

Comment:
I think we'll need to add tracking here but that's good for a follow up PR.

Reply:
#13394 does that :+1:

Reply:
Actually, it only does in case we're using the Stripe Connect Nudge, but not if we're still using the in-block connect button :slightly_frowning_face: